### ISSUE TITLE
Endless Key app v1.0.51.0's description

### ISSUE THREAD

**Issue by starnight** [endlessm/kolibri-app#115]
We got a new report about Endless Key app (v1.0.51.0) from MS store after we promote it from Alpha Test to stable.

Here is the report's content:

### Please take the following action
We reviewed your submission and identified some changes that are needed before we can publish or update the app. Please make these changes and resubmit your app. For more information, contact your Microsoft representative. For faster responses, please include your Product ID.

### Report details

| Content requirement policies | Notes to publisher |
-- | --
| 10.1.4.3 App Quality - Description | Please provide a description that’s clear and is about your app. Tips on writing app descriptions are available at https://docs.microsoft.com/en-us/windows/uwp/publish/write-a-great-app-description Found in: English |

**Comment by starnight** [comment]
Here is the current app's description: "Learning and discovery resources. Download once and use offline. (Beta version)"

![image](https://github.com/endlessm/endless-key-app/assets/2100946/132d0af4-8e73-443a-a423-be9e1f900e1a)

@erikos @jofilizola Should we update app's description again?

**Comment by starnight** [comment]
To make it submit successfully, I find the original "Endless Key - App Information" sheet from EOSF drive.  Take the short description `The Endless Key robust, engaging and safe digital learning environment, with high-quality learning and discovery resources that can be accessed offline.` appended with `(Beta version)` string.

Then, submit Endless Key app to MS store again.  It is in certification again now.

**Comment by wjt** [comment]
There is a much better description on https://play.google.com/store/apps/details?id=org.endlessos.Key&pli=1

> An infinite stream of discovery, inspiration, empowerment, and learning awaits you with the Endless Key App!
>
> Designed for all learners, and accessible to those with no or limited internet, the Endless Key lets you explore at your own pace, expand on your classroom lessons, and push your knowledge to new heights.
> 
> Learners can explore dozens of educational resources that are thoughtfully curated to provide extended learning, academic enrichment, and interest-based options like coding, arts & crafts, cooking, and career exploration, no internet required! The app is free to download. To learn more, visit our website endlessos.org/key.

Can't we make it match?

**Comment by erikos** [comment]
1.0.51.0 is available on public MS store now.  We can open another ticket, if there is another new report.
